index.html

```diff
@@ -40,31 +51,233 @@ <h1 class="heading_second">
+        </div>
+      </div>
+    
+    </div>
+
+
+    <div class="card_container5">
+      <div class="work_card">
+        <div class="img_cardiv">
+          <img class="mobi" src="./images/Snap4.png" alt="icon" />
+        </div>
+        <div class="side_info">
+          <h3 class="project_name">
+            Multi-Post Stories
+          </h3>
+          <ul class="first_list">
+            <li>CANOPY</li>
+            <li>Back End Dev</li>
+            <li>2015</li>
+          </ul>
+          <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+          <ul class="tech_list">
+            <li>html</li>
+            <li>css</li>
+            <li>javaScript</li>
+          </ul>
+          <button class="btn1" type="button">See Project</button>
+        </div>
+      </div>
+    
+    </div>
+ 
+
+
+    <section id="secondlast_section">
+      <div class="section-last-conatiner">
+        <div class="home_containe">
+          <div>
+            <h1 class="aboutdiv">About<br>Myself</h1>
+          </div>
+          <div class="first-para">
+          <p>Hello I'm a software developer! I can help you build a product , feature or website Look through some of my
+            work and experience! If you like what you see and have a project you need coded, don't hestiate to contact
+            me.
+          </p>
+        </div>
+          <span>LET'S CONNECT</span>
+            <ul class="social_links">
+            <li>
+              <a href="#">
+                <img src="./images/Vector-2.jpg" class=logo-icons alt=union>
+              </a>
+            </li>
+            <li>
+              <a href="#">
+                <img src="./images/Linkedin-icon.jpg" class=logo-icons alt=union>
+              </a>
+            </li>
+            <li>
+              <a href="#">
+                <img src="./images/Vector-3.jpg" class=logo-icons alt=union>
+              </a>
+            </li>
+            <li>
+              <a href="#">
+                <img src="./images/Vector-4.jpg" class=logo-icons alt=union>
+              </a>
+            </li>
+            <li>
+              <a href="#">
+                <img src="./images/Vector.jpg" class=logo-icons alt=union>
+              </a>
+            </li>
+          </ul>
+          <button class="btn1" type="button">Get My Resume</button>
+        </div>
+        <div>
+          <div>
+            <div class="languages_logo">
+              <h4 class="languages_heading">Languages</h4>
+              <div class="image_float">
+                <img src="./images/IconChevronDownLarger.png" alt="icon">
+              </div>
+            </div>
+            <ul class="languages">
+              <li class="ellipse1">
+                <img src="./images/Ellipse_j.png" alt="icon">
+                <span>JavaScript</span>
+              </li>
+              <li class="ellipse1">
+                <img src="/images/Ellipse_h.png" alt="icon">
```

Comment:
## OPTIONAL 

- [ ] Kindly fit the broken image link in your project 👍🏿 
![Uploading Screenshot_47.png…]()

Reply:
please check what my browser show me
http://127.0.0.1:5501/index.html